lighthouse-core/runner.js

```diff
@@ -234,6 +234,8 @@ class Runner {
         gatherMode: undefined,
         auditMode: undefined,
         output: undefined,
+        budgetsPath: undefined,
+        budgetsJSON: undefined,
```

Comment:
is this needed here? I don't believe it should be...

Reply:
I added this because `artifacts.settings` includes budgetsPath (but not budgetsJSON) while `settings` includes budgetsJSON (but not budgetsPath), so it was throwing bc audit/gather settings didn't match.

Is this the correct configuration of the budgetsPath/budgetsJSON flags on settings/artifacts.settings? What's currently being set matches how I interpreted the flags should work, but Lighthouse flag convention isn't really my area of expertise :)

Reply:
Well for the error I *think* that's because you've updated artifacts.json with the first iteration of this PR but not the current state (see comment below)

definitely want to keep `budgetsJSON` in here though @khempenius  :)

It's an awesome feature to be able to re-run the same gathered artifacts with a new budget!